lib/compat/wordpress-6.1/get-global-styles-and-settings.php

```diff
@@ -0,0 +1,101 @@
+<?php
+/**
+ * API to interact with global settings & styles.
+ *
+ * @package gutenberg
+ */
+
+/**
+ * Returns the stylesheet resulting of merging core, theme, and user data.
+ * This is a duplicate of wp_get_global_stylesheet
+ *
+ * @since 5.9.0
+ *
+ * @param array $types Types of styles to load. Optional.
+ *                     It accepts 'variables', 'styles', 'presets' as values.
+ *                     If empty, it'll load all for themes with theme.json support
+ *                     and only [ 'variables', 'presets' ] for themes without theme.json support.
+ *
+ * @return string Stylesheet.
+ */
+function wp_get_global_stylesheet_gutenberg( $types = array() ) {
```

Comment:
This function is almost a copy of what we have in core

Reply:
Is this any different to `gutenberg_get_global_stylesheet()` in [lib/compat/wordpress-6.0/get-global-styles-and-settings.php ](https://github.com/WordPress/gutenberg/blob/848db705d6436bc267234910eb653968e6bf2893/lib/compat/wordpress-6.0/get-global-styles-and-settings.php#L76)?

I think the general approach is to copy that function over to the new compat file and keep the `gutenberg_` naming convention.